bindata/v4.0.0/controller/clusterrole.yaml

```diff
@@ -24,13 +24,22 @@ rules:
   - watch
   - update
   - patch
+- apiGroups:
+  - apiregistration.k8s.io
+  resources:
+  - apiservices
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
 - apiGroups:
   - ""
-  resourceNames:
-  - extension-apiserver-authentication
   resources:
   - configmaps
   verbs:
   - get
   - list
   - watch
+  - update
```

Comment:
no create?

Reply:
it only injects the cert IIRC, create should not be needed

Reply:
The tests are passing, and configmap injection is well-tested.